util/stmtsummary/statement_summary.go

```diff
@@ -230,6 +230,26 @@ func (ssMap *stmtSummaryByDigestMap) ToDatum() [][]types.Datum {
 	return rows
 }
 
+// GetMoreThanOnceSelect gets select SQLs that occurred more than once.
+func (ssMap *stmtSummaryByDigestMap) GetMoreThanOnceSelect() ([]string, []string) {
+	ssMap.Lock()
+	values := ssMap.summaryMap.Values()
+	ssMap.Unlock()
+
+	schemas := make([]string, 0, len(values))
+	sqls := make([]string, 0, len(values))
+	for _, value := range values {
+		summary := value.(*stmtSummaryByDigest)
+		summary.Lock()
+		if strings.HasPrefix(summary.normalizedSQL, "select") && summary.execCount > 1 {
```

Comment:
Out of curiosity, can we make the SQL occurrent number under control instead of `1`?

Reply:
I guess we can, but we can do it when only when it is necessary because doing it would require another session variables, for now, it is enough.